### DISCUSSION TITLE
How to export a model glb with a texture + Alpha (3ds Max) ?

### DISCUSSION THREAD

**Discussion by REOREV** [google/model-viewer#2444] [Q&A]
### Description

Hello,
I tried to generate a glb from 3ds max with a bitmap + alpha for the transparence (a simple teapot, material steel, logo png). 
For that, I used a PNG with an alpha without succes. I Tried too an blend (logo + mask) but I can't have the transparence.
How can I generate a model (standard or physical materials) and add simple logo png for example ?
Thank you.
Frank

#### Live Demo

https://test-alpha-3d.glitch.me

### Version

- model-viewer: vX.X.X

### Browser Affected

- [ x] Edge
- [x ] Firefox

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [x ] Windows

### AR

- [ ] WebXR
- [ x] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
This is a 3DS Max question, so you might have better luck on their forums. It could be a GLB export bug.
